config/enabled.yml

```diff
@@ -1338,6 +1338,12 @@ Lint/RedundantWithObject:
   Description: 'Checks for redundant `with_object`.'
   Enabled: true
 
+Lint/RegexpInCondition:
+  Description: >-
+                 Do not use regexp literal in condition.
```

Comment:
Perhaps the message should be more specific, since e.g. `if /foo/ =~ str` is also a regexp literal in a condition. Perhaps the cop name should even be `RegexpCondition`? 🤔

Reply:
Guess @pocke just followed the (bad) naming of the other cop. Maybe it should have been named `LiteralAsCondition` or `LiteralCondition`.

Reply:
Ah. yes.